ci/test/00_setup_env_i686_multiprocess.sh

```diff
@@ -6,11 +6,12 @@
 
 export LC_ALL=C.UTF-8
 
-export CONTAINER_NAME=ci_native_multiprocess
+export HOST=i686-pc-linux-gnu
+export CONTAINER_NAME=ci_i686_multiprocess
 export DOCKER_NAME_TAG=ubuntu:20.04
-export PACKAGES="cmake python3 python3-pip llvm clang"
-export DEP_OPTS="DEBUG=1 MULTIPROCESS=1"
+export PACKAGES="cmake python3 python3-pip llvm clang g++-multilib"
+export DEP_OPTS="DEBUG=1 MULTIPROCESS=1 CC='clang -m32' CXX='clang++ -m32'"
```

Comment:
I believe this won't work due to #22184.

Reply:
It should still compile successfully. Though, I removed it now to minimize the diff

Reply:
> It should still compile successfully.

Yes, it should. But with gcc, not clang.